task_processing/plugins/mesos/execution_framework.py

```diff
@@ -423,16 +423,17 @@ def resourceOffers(self, driver, offers) -> None:
 
         with self._lock:
             if self.task_queue.empty():
-                if not self.are_offers_suppressed:
```

Comment:
You can get rid of this variable now, right?

Reply:
are_offers_suppressed is still used in enqueue_task.

Reply:
Ah! ship-it